### ISSUE TITLE
Convert ValuePanel to AccordionBox.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/unit-rates#236]
Related to https://github.com/phetsims/sun/issues/919#issuecomment-2546357326 ...

ValuePanel should be converted to an AccordionBox, important for PhET-iO and a11y features.  Screenshots below show where ValuePanel appears in the sim.

There is a note about this at the top of ValuePanel.ts:

```ts
NOTE: While this seems like a good application for AccordionBox,
I ran into problems related to justifying and i18n.
```

This issue is deferred until work resumes on this sim.

<img width="469" alt="screenshot_3655" src="https://github.com/user-attachments/assets/9f763177-6e4b-4395-b70d-8d01e45a0e84" />

<img width="213" alt="screenshot_3656" src="https://github.com/user-attachments/assets/9598da46-26f0-4249-895c-9060c93ec25f" />
